legacy/app/data/post-create/post-editor.directive.js

```diff
@@ -132,8 +132,11 @@ function PostEditorController(
                             attr.value.value = dayjs(attr.value.value).format('YYYY-MM-DD');
                         } else if (attr.default) {
                             try {
-                                attr.value.value = dayjs(new Date(attr.default).format('YYYY-MM-DD'));
-                            } catch (err) {
+                                let defaultValue = dayjs(new Date(attr.default)).format('YYYY-MM-DD');
+                                // Safeguarding against invalid default dates below. We should add validation in the survey setup instead
+                                if (defaultValue !== 'Invalid Date') {
```

Comment:
Are we reasonably confident this is going to be "Invalid Date" regardless of environment, new versions etc.. ?  Is there something easily available but also less likely to change? (i.e. an object type or property)

Reply:
Good point. Changed this a bit.